testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -705,6 +705,19 @@ QUnit.module('functionality', moduleSetup, () => {
         assert.equal(displayValue, 'zero', 'value is rendered correctly');
     });
 
+    // T1135997
+    QUnit.test('Show selectBox in fullScreen mode', function(assert) {
```

Comment:
1. We usually add a ticket ID as a test name's suffix.
2. Test name should answer on a question "how should it work"

```suggestion
    QUnit.test('can be rendered with a dropDown in fullScreen mode (T1135997)', function(assert) {
```